bms_temperature_handler.py

```diff
@@ -0,0 +1,28 @@
+SUPPORTED_TEMPERATURE_UNITS = ['Celsius', 'Fahrenheit', 'Kelvin']
+KELVIN_CONSTANT = 273.15
+charging_temperature_unit = None
```

Comment:
Global variables make it not-thread-safe. Can we avoid this, returning the unit from `set_charging_temperature_unit` and pass the return value to `perform_temperature_conversion_to_celsius`